Bottom: 72274864ab2fa3553d2cdfb306dae420573ff3ee
Top:    8ff65189091641d19d11068c8daa4240680ec6fe
Author: scott snyder <sss@karma>
Date:   2020-04-05 00:13:30 -0400

Refresh of cad-cah-smbc

---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/core/Comic.java b/ComicReader/src/com/blogspot/applications4android/comicreader/core/Comic.java
index a7f4155..7473e7e 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/core/Comic.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/core/Comic.java
@@ -550,20 +550,14 @@ public abstract class Comic extends ComicParser {
 	 * @return next strip
 	 */
 	public Strip getNextStrip() {
-          Log.d("zzz", "getNextStrip1");
 		if (isCurrentLatestStrip()) {
 			return mCurrent;
 		}
-          Log.d("zzz", "getNextStrip2");
 		if (!mCurrent.hasNext()) {
-                  Log.d("zzz", "getNextStrip3");
 			mCurrent.setNext(getNextStripUrl());
 		}
 		String uid = mCurrent.uid();
-                Log.d("zzz", "getNextStrip4 " + uid);
-                String next = mCurrent.getNext();
-                Log.d("zzz", "getNextStrip5 " + next);
-		Strip s = _querySetCurrentUid(next);
+                Strip s = _querySetCurrentUid(mCurrent.getNext());
 		s.setPrevious(uid);
 		return s;
 	}
